ksqldb-streams/src/test/java/io/confluent/ksql/execution/streams/SourceBuilderV1Test.java

```diff
@@ -1055,7 +1051,8 @@ private void givenWindowedSourceStream(final int pseudoColumnVersion) {
         OptionalInt.of(pseudoColumnVersion)
     );
   }
-  private void givenWindowedSourceStream() {
+  private void
+  givenWindowedSourceStream() {
```

Comment:
nit: whitespace change.

Reply:
I was actually referring to the fact that this line was split between two lines but your other change is good too 😆

Reply:
I don't understand how I missed this one o.O